src/zenml/zen_server/routers/model_versions_endpoints.py

```diff
@@ -256,6 +256,26 @@ def delete_model_version_artifact_link(
     )
 
 
+@router.delete(
+    "/{model_version_id}" + ARTIFACTS,
+    responses={401: error_response, 404: error_response, 422: error_response},
+)
+@handle_exceptions
+def delete_all_model_version_artifact_link(
```

Comment:
```suggestion
def delete_all_model_version_artifact_links(
```

Should have an `s`? (And change whatever else needs changing downstream of this change?)